Modules/posixmodule.c

```diff
@@ -4747,6 +4747,10 @@ os_listdrives_impl(PyObject *module)
     return result;
 }
 
+#endif /* MS_WINDOWS_DESKTOP || MS_WINDOWS_SYSTEM */
+
+#if defined(MS_WINDOWS_APP) || defined(MS_WINDOWS_SYSTEM)
```

Comment:
IIRC, _APP is more widely available than _DESKTOP, which means some users will be able to `listvolumes` but not drives or mount points? Is there any value to this? Or should we just mark them all as requiring _DESKTOP?

(If I'm wrong and it's the other way around, then my question becomes "what's the point in providing `listmounts` if the function to get the volume names isn't available)

Reply:
I just used the guards that were used in the windows headers providing the underlying functionality.

Reply:
Well, I guess someone somewhere found some value, so we may as well just mirror them and not exert ourselves too much.

Reply:
That was my thought as well :smile: